src/features/section/ui/SectionForm.tsx

```diff
@@ -2,20 +2,22 @@ import { Alert, Button, FloatingLabel, Form } from 'react-bootstrap';
 import sectionsQueries from '../queries';
 import { useState } from 'react';
 import { parseSectionData } from '../service/utils';
+
 interface SectionFormProps {
   closeModal: () => void;
 }
+
 export default function SectionForm({ closeModal }: SectionFormProps) {
-  const { mutate: createMutate } = sectionsQueries.create();
+  const { mutate: createMutate } = sectionsQueries.createSection();
   const [errorMessage, setErrorMessage] = useState('');
 
   const handleMutate = (e: React.FormEvent<HTMLFormElement>) => {
```

Comment:
```suggestion
  const handleMutate = React.FormEventHandler<HTMLInputElement> = e => {
```
매개변수 `e` 에 `FormEvent`를 직접 할당하는 것은 `e`의 타입을 명시적으로 표시하기 때문에 직관적이지만, 함수 전체의 타입을 한 눈에 알아보기 힘들 수 있다고 합니다.

실제로
![image](https://github.com/user-attachments/assets/9cf19994-b673-4071-bd35-a5845e4d1521)
`onSubmit` 에 마우스를 올려보면 나오는 타입이 Handler타입입니다. 
저는 그래서 이벤트 핸들러를 작성할 때 해당 형식을 많이 사용합니다.

그래서 함수선언시 `FromEventHandler`를 사용해 좀더 명확한 함수 타입을 지정할 수 있죠


https://github.com/palantir/blueprint/issues/3372#issue-412400282
해당 issue는 위에 제가 언급해드린 내용에 대해 논의한 내용입니다.

Reply:
```ts
 const handleMutate: FormEventHandler<HTMLFormElement> = e => {
    e.preventDefault();

    const formData = new FormData(e.currentTarget);
    const quizData = Object.fromEntries(formData.entries());
    const parsedQuizData = parseQuizData(quizData);

//...생략
```
다음과 같이 작성할 수 있다는 이야기인가요? 확실히 함수의 타입을 만들어놓으면 인자에 개별타입을 지정해주지 않아도 되니 유용할 것 같네요 좋은 레퍼런스 감사합니다👍

Reply:
@zzzRYT abea9c1fa63fe5eb276d368238168897833019ce ,[89fc21b](https://github.com/modern-agile-team/coko-admin/pull/31/commits/89fc21b28c0f1a649dcc3b474b5ab69f18940c11)
덕분에 좋은 아이디어 얻어서 이번에 만든 훅에도 적용 시켜 봤습니다!